src/xmipp/libraries/py_xmipp/deepConsensusWorkers/deepConsensus_deepLearning1.py

```diff
@@ -73,6 +73,38 @@ def writeNetShape(netDataPath, shape, nTrue, nModels):
       os.makedirs(netDataPath )
     with open(netInfoFname, "w" ) as f:
         f.write("inputShape: %d %d %d\ninputNTrue: %d\nnModels: %d" % (shape+(nTrue, nModels)))
+
+def writeNetAccuracy(netDataPath, val_acc):
+  '''
+    netDataPath= self._getExtraPath("nnetData")
+  '''
+  netAccFname = os.path.join(os.path.dirname(netDataPath), "netValAcc.txt")
+  if not os.path.exists(netDataPath):
+    os.makedirs(netDataPath)
+  with open(netAccFname, "w") as f:
+    f.write("val_acc: %f" %val_acc)
+
+def loadANDwriteNetAccuracy(netDataPath, nModels):
+  '''
+      netDataPath= self._getExtraPath("nnetData")
+  '''
+  list_Acc = []
+  for n in range(nModels):
+    checkPointsName = os.path.join(netDataPath, "tfchkpoints_%d"%n)
+    valAccFn = os.path.join(checkPointsName, "netValAcc.txt")
+    with open(valAccFn) as f:
+      lines = f.readlines()
```

Comment:
this will read whole file, but on the next line you're accessing only the first line of the file -> wasteful